RELEASE_NOTES.md

```diff
@@ -1,5 +1,80 @@
+    - feat: add ingress configuration for Thanos sidecar, enabling external access from a centralized thanos querier running in another cluster
+    - feat: add scrape timeout config to service monitor to avoid timeouts on slow kubelets
+    - feat: add docker checksum option to improve security for deployed containers
+    - feat: add option to disable availability rules
+    - feat: enable scraping /metrics/resource for kubelet service
+    - feat: [prometheus] enable namespace overrides
+    - feat: [prometheus] allow additional volumes and volumeMounts
+    - feat: [alertmanager] add volume and volume mounts to spec
+    - feat: [alertmanager] add support for serviceAccount.annotations
+    - feat: [grafana] enable adding annotations to all default dashboard configmaps
+    - chore: bump prometheus to v2.18.2
+    - chore: bump alertmanager to v0.21.0
+    - chore: bump hyperkube to v1.16.12
+    - chore: bump grafana to v5.3.0
+    - fix: add missing grafana annotations to k8s-coredns dashboard
+    - fix: reduced CPU utilization and time lag for code_verb:apiserver_request_total:increase30d scrape
+    - fix: invalid image pull policy for the admission webhook patch
+    - fix: alert "KubeNodeUnreachable" no longer fires on an autoscaling scale-down event ([#444](https://github.com/mesosphere/kubernetes-base-addons/pull/444), [@samvantran](https://github.com/samvantran))
+  - disable ServiceMonitors for kube-controller-manager and kube-scheduler. kubernetes has determined the ports that were used for these tests was insecure and has limited it to localhost only. This causes these specific tests to fail. The state of the controller-manager and scheduler pods are still tracked in general as pods. ([#474](https://github.com/mesosphere/kubernetes-base-addons/pull/474), [@dkoshkin](https://github.com/dkoshkin))
+
+## stable-1.15-2.2.0, stable-1.16-2.2.0, stable-1.17-2.2.0
+
+* Prometheus
+  * Fix an issue that may cause Grafana's home dashboard to be empty. ([#351](https://github.com/mesosphere/kubernetes-base-addons/pull/351), [@branden](https://github.com/branden))
+  * disable ServiceMonitors for kube-controller-manager and kube-scheduler. kubernetes has determined the ports that were used for these tests was insecure and has limited it to localhost only. This causes these specific tests to fail. The state of the controller-manager and scheduler pods are still tracked in general as pods. ([#474](https://github.com/mesosphere/kubernetes-base-addons/pull/474), [@dkoshkin](https://github.com/dkoshkin))
+  * Improve Grafana dashboard names and tags for dashboards tied to addons ([#352](https://github.com/mesosphere/kubernetes-base-addons/pull/352), [@gracedo](https://github.com/gracedo))
+* Traefik
+  * fix metrics access and reporting ([#349](https://github.com/mesosphere/kubernetes-base-addons/pull/349), [@gracedo](https://github.com/gracedo))
+
 ## stable-1.15-2.0.0, stable-1.16-2.0.0, stable-1.17-2.0.0
```

Comment:
Shall we add 2.1.0 notes as well?

Reply:
oops, apparently we didn't have them in 2.2.0 :/